src/explore-education-statistics-common/src/modules/find-statistics/components/SummaryRenderer.tsx

```diff
@@ -77,9 +78,18 @@ export default function SummaryRenderer({
                   <p className="govuk-body-s">{dataSummary[index]}</p>
                 )}
               </div>
-              <Details summary={`What is ${meta.indicators[key].label}?`}>
-                Overall absence is the adipisicing elit. Dolorum hic nobis
-                voluptas quidem fugiat enim ipsa reprehenderit nulla.
+              <Details summary={`Define '${meta.indicators[key].label}'`}>
+                <p>
+                  {`${
+                    meta.indicators[key].label
+                  } is the adipisicing elit. Dolorum hic nobis voluptas quidem fugiat enim ipsa reprehenderit nulla.`}
```

Comment:
I'm suspecting we might need to add a api property for this text as there might be an expectation of it being contextual rather than an example for unmoderated testing.

Reply:
I was going on what Laura was wanting. She said to hard-code some example in there.

Reply:
Cool, not a problem then if that's the case.